### ISSUE TITLE
`@tabler/icons-svelte` icons `size` and `stroke` props typed incorrectly

### ISSUE THREAD

**Issue by emmnull** [tabler/tabler-icons#801]
The `IconProps` interface is typed incorrectly and ends up being too zealous with only accepting number values for `size` and `stroke`. Looking at how the props are used within the components, passing string values to either of these should be considered valid. For example:

```svelte
<IconMoon size="1em" stroke="2px" />
```

consists of valid prop values in the scope of SVG attributes, and actually works as is although it makes typescript cry no-no.

**Comment by christoph-jeanluc-schneider** [comment]
Hotfix: [@abineo/svelte-icons](https://www.npmjs.com/package/@abineo/svelte-icons).
Should be a drop in replacement.

**Comment by iolyd** [comment]
@christoph-jeanluc-schneider good idea with that fork. I ended up switching to https://github.com/lucide-icons/lucide due to the standing issue with dev/build slowdowns with tabler-icons.
